database/etcddb/database.go

```diff
@@ -0,0 +1,45 @@
+package etcddb
+
+import (
+	"context"
+	"time"
+
+	"github.com/aptly-dev/aptly/database"
+	clientv3 "go.etcd.io/etcd/client/v3"
+)
+
+var Ctx = context.TODO()
+
+func internalOpen(url string) (*clientv3.Client, error) {
+	cfg := clientv3.Config{
+		Endpoints:            []string{url},
+		DialTimeout:          30 * time.Second,
+		MaxCallSendMsgSize:   (2048 * 1024 * 1024) - 1,
+		MaxCallRecvMsgSize:   (2048 * 1024 * 1024) - 1,
+		DialKeepAliveTimeout: 7200 * time.Second,
+	}
+
+	cli, err := clientv3.New(cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	return cli, nil
```

Comment:
We could replace this with `return clientv3.New(cfg) since the library returns a `nil` client in case of error

Reply:
done